### ISSUE TITLE
Time sorting is broken

### ISSUE THREAD

**Issue by RAMAR-RAR** [NimaKhajehpour/MyMood#37]
Following the insertion of a mood with a 24-hour time format (e.g. 13:50), the single mood will be saved as PM, but when displayed alongside other moods, it will be sorted as if it were AM.

**Comment by UnrivaledScapegoat** [comment]
My previous screenshots show the problem with the time sorting in both graphs and in normal view:
![Screenshot_20231120-152501_MyMood](https://github.com/NimaKhajehpour/MyMood/assets/151566791/6edf433f-767c-4ebf-92bd-9afb598e7598)

![Screenshot_20231115-080040_MyMood](https://github.com/NimaKhajehpour/MyMood/assets/151566791/13f22508-7bf1-4f99-84f3-9d370f7e3593)

**Comment by UnrivaledScapegoat** [comment]
But after I tried to reproduce this issue just now at 00:34, time sorting is magically fixed only for this demo. However, the previous days still show that time sorting is broken.
![Screenshot_20231121-003826_MyMood](https://github.com/NimaKhajehpour/MyMood/assets/151566791/f4608db6-8383-4a52-b01d-332be9f6927a)

**Comment by NimaKhajehpour** [comment]
hey thank you very much for using the app. i know that i haven't maintained the app for a long time and this issue is open for nearly a year. but i am resuming the maintenance of the application and re-designing it. the app going to be an online app too. so there are many stuff going on.
i will also look into the bug you have mentioned.

**Comment by UnrivaledScapegoat** [comment]
> the app going to be an online app too.

Nooo. Please keep it as offline app. I am very concious of my mood data being hijacked for unwanted ad targeting. Also users will desire privacy when it comes to psychological aids like this app.

**Comment by NimaKhajehpour** [comment]
yup i heard you everyone said that they prefer it to be offline the app will always stay offline. no worries at all.
hope you enjoy the app 😉
